customers.go

```diff
@@ -135,6 +135,11 @@ type CustID struct {
 	CustomerUUID   string `json:"customer_uuid,omitempty"`
 }
 
+// DeleteCustomerInvoicesParams - optional param for deleting all customer invoices.
+type DeleteCustomerInvoicesParams struct {
+	CustomerExternalID string
```

Comment:
It would probably better if if you put all the parameters in the struct e.g. DataSourceUUID and CustomerUUID.

Reply:
@hassansin see previous versions of PR & comments. If Ishan does that, then he also needs to check for presence, otherwise invalid URLs can be generated.

Reply:
I think of it as equivalent in ruby `method(string, string, keyword_arg: nil)`